doc/api/path.md

```diff
@@ -318,8 +318,9 @@ The `path.normalize()` method normalizes the given `path`, resolving `'..'` and
 `'.'` segments.
 
 When multiple, sequential path segment separation characters are found (e.g.
-`/` on POSIX and `\` on Windows), they are replaced by a single instance of the
-platform specific path segment separator. Trailing separators are preserved.
+`/` on POSIX and either `\` or `/` on Windows), they are replaced by a single 
+instance of the platform specific path segment separator. Trailing separators 
```

Comment:
> segment separator, [`path.sep`](https://nodejs.org/api/path.html#path_path_sep).

windows has 2, the text above would allow normalize to replace all backslashes with a forwardslash on windows :-)

I suggest